app/utils/database.py

```diff
@@ -1270,12 +1280,41 @@ def migrate_5_to_6():
                 auth.save()
             migrate(DATABASE_MIGRATOR.drop_column("hbuser", "password_hash"))
         SchemaHistory.insert(  # pylint: disable=E1120
-            version=5,
+            version=6,
             comment="Remove password_hash from hbuser",
             applied=datetime.now(),
         ).execute()
 
 
+def migrate_6_to_7():
+    """
+    Migrate between schema version 5 and 6.
```

Comment:
probably a wrong comment here, with regard to the function's name?

Reply:
Yes, :eyes: .